proto/validate/validate.proto

```diff
@@ -0,0 +1,862 @@
+syntax = "proto2";
```

Comment:
isnt it possible to define this dependency in buf.gen.yaml ?

Reply:
I think it already is, but protoc gave an error.

Reply:
OK, this is bad, we should also check if docs.json can be built with `buf`